mlcomp/parallelm/extra/sagemaker/monitor/job_monitor_base.py

```diff
@@ -51,10 +57,40 @@ def set_on_complete_callback(self, on_complete_callback):
         self._on_complete_callback = on_complete_callback
         return self
 
+    def _total_running_time_sec(self, describe_response):
+        create_time = self._job_create_time(describe_response)
+        if create_time is None:
+            return None
+
+        end_time = self._job_end_time(describe_response)
+        if end_time:
+            return (end_time - create_time).total_seconds()
+        else:
+            return (datetime.now(pytz.UTC) - create_time).total_seconds()
```

Comment:
Is UTC the correct time Zone?

Reply:
Yes. It is always best to work with UTC and SageMaker times are UTC.